root/static/scripts/annotation/AnnotationHistoryTable.js

```diff
@@ -101,8 +101,9 @@ const AnnotationHistoryTable = ({
                 <td>
                   <input
                     className="old"
+                    data-index={index}
                     defaultChecked={index === 1}
-                    disabled={annotation.id >= state.selectedNew}
+                    disabled={index <= state.selectedNew}
```

Comment:
Same here and below.

Reply:
The whole idea is that rather than the annotation id (which doesn't guarantee sorting by time, probably because of some pre-NGS issues), we just use the index of the annotation in the list, since that's sorted by date anyway. Seemed easier than actually doing the date comparisons.

Reply:
The unclear bit here was the change of direction in the comparison. IIUC, annotation IDs were mostly sorted by ascending date, while indexes are strictly sorted by descending date. If correct, the commit message should probably mention it.

Reply:
Yeah, that's the point, since they're sorted from new to old, the sort here changes :) I can add it to the message.